samcli/commands/local/invoke/cli.py

```diff
@@ -100,3 +100,16 @@ def _get_event(event_file_name):
     # accidentally closing a standard stream
     with click.open_file(event_file_name, 'r') as fp:
         return fp.read()
+
+
+def _get_debug_context(debug_port, debug_args, debugger_path):
+    """
+    Returns a debug context from debug options; Separated out for unit testing.
+    :param int debug_port: Container debug port
+    :param string debug_args: Extra debug arguments for process
+    :param string debugger_path: Path to debugger on host
+    :return DebugContext:
+    """
+    if not debug_port or debug_args or debugger_path:
```

Comment:
you don't need this method anymore, do you?

Reply:
I do not.